docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #69.                    D = core._mass(Q, S, QS, T_sliding_mu[i], T_sliding_std[i], 
</pre><p>This would then be:</p><p> </p><pre class="ql-syntax" spellcheck="false">D = core._mass(Q, T[start:stop+m-1], M_T[i], Σ_T[i], M_T[start : stop], Σ_T[start : stop])
</pre><p> </p><p>Which I find very readable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>